### PR TITLE
Portfolio: validate contact form

### PR DESCRIPTION
**Pull request by Allenkeys**
## Project requirements

- You need to implement a simple validation:

  - The content of the email field has to be in lower case.
- You should implement the following interactions:

  - When the user submits the form, you check if the email is in lower case.
  - If the validation is OK, the form is sent.
  - If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

### CODE REVIEW

**Review by tresorsawasawa — COMMENTED** [review]
# STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @ ,:wave: :wave: 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 🛑

You have submitted a project that has some linters errors and that also requires **pair-programming** with your partners. Kindly make sure your partner has some contributions on your pull request and fix those liners errors before requesting a new review-request.
Screenshot from 2022-09-09 16-13-44



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

---
Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

---
_Invalid Code Review Request **Does not count** into the code reviews limit._

File: js/main.js [hunk 1]
```diff
@@ -22,3 +27,51 @@ navLinks.forEach((navLink) => {
...
+  const emailValue = email.value.trim();
+  const fnameValue = fname.value.trim();
+  const messageValue = message.value.trim();
+  let isFalse = false;
+
+  if (/^[a-z0-9._-]+@[a-z0-9.-]+\.[a-z]{2,4}$/g.test(emailValue)) {
+    setSuccess(email);
+    isFalse = true;
+  }
+  setError(email, 'Invalid email');
```
Comment (mwanawabangona) [inline js/main.js]: - [ ] `required`
- [ ] Good so far! 👏 
- [ ]  as the stated in the requirements.
- [ ]  If the validation is not OK, you show an error message to the user `near the submit button` informing them of the error and the form is not sent.
-  kindly move the error message near the submit button. 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Allenkeys ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Allenkeys** [comment]
@tresorsawasawa  I don't know what changes to be made.
I've re-submitted for review, it got turned down with no feedback on what next to do.

**Review by medaminedev66 — APPROVED** [review]
Hi @Allenkeys  and @Zuhaib042 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- The client-side validation works as expected
- Your code is clean.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**